src/print.cc

```diff
@@ -398,8 +398,8 @@ static void print_text_menu(GtkWidget *box, PrintWindow *pw)
 
 	hbox = pref_box_new(subgroup, FALSE, GTK_ORIENTATION_HORIZONTAL, PREF_PAD_BUTTON_GAP);
 
-	button = pref_button_new(nullptr, GTK_STOCK_SELECT_FONT, _("Font"), FALSE,
-				 G_CALLBACK(print_set_font_cb), const_cast<char *>("Image text font"));
+	button = pref_button_new(nullptr, GQ_ICON_SELECT_FONT, _("Font"),
+				 G_CALLBACK(print_set_font_cb), (gpointer)"Image text font");
```

Comment:
Probably should stay with c++ cast.
```suggestion
				 G_CALLBACK(print_set_font_cb), const_cast<char *>("Image text font"));
```